### DISCUSSION TITLE
What is the best way to customize DateInput dropdown?

### DISCUSSION THREAD

**Discussion by BOHDANff** [mantinedev/mantine#5737] [Q&A] [answered]
Hello, I want to customize DateInput dropdown by adding 'Today' button under the calendar which will move date to today. Should I create custom DateInput from zero in such case or there is any better way to realize it?

**Comment by rtivital** [comment, marked as answer]
Currently, this feature is not supported. You can build a custom component for that.

**Reply by BOHDANff** [reply]
Thank you for the answer
